src/components/css/Calculator.css

```diff
@@ -0,0 +1,70 @@
+
+:root {
+  --orange: orange;
+  --lightGray: rgb(241, 241, 241);
+  --gray: rgb(191, 189, 189);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  width: 90vw;
+  height: 100vh;
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calculator {
+  width: 90vw;
+  height: 100vh;
+  margin: 5% auto;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.3);
+}
```

Comment:
- [`OPTIONAL`] Great job so far. I would suggest you take a look at the [**big picture**](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/images/math_magicians_calculator.png) while building your project. While building the whole wireframe is not required in this milestone it's important that you consider it while building other parts of the UI. 
  - Looking at your calculator now, you have set a `height: 100vh` and `width: 90vw` making the calculator cover the entire screen. While it's okay since you don't have any other content on the page I would advise that you reduce its size to make it look more like a calculator 😊 in terms of its size as it's currently looking a bit off. 
  
  Happy coding! 😊🚀🍀